openbb_terminal/helper_funcs.py

```diff
@@ -150,8 +150,6 @@ def parse_and_split_input(an_input: str, custom_filters: List) -> List[str]:
     if an_input:
         if an_input == "/":
             an_input = "home"
-        elif an_input[0] == "/":
-            an_input = "home" + an_input
```

Comment:
Hi @DidierRLopes, can you comment on this addition? I've noticed that when I run routine files what it does is the following. Let's say I have the following file:

```
# Go into the stocks context
stocks

# Load a company ticker, e.g. Apple
load AAPL

# Show a candle chart with a 20 day Moving Average
candle --ma 20

# Switch over to the Due Diligence menu
dd

# Show analyst ratings
analyst

# Show price targets charts
pt

# Show future estimations
est

# Return to home
home
```

This is translated in the function as `/stocks/load AAPL/candle --ma 20/dd/analyst/pt/est/home`. However, given that piece of code it is adjusted to `home/stocks/load AAPL/candle --ma 20/dd/analyst/pt/est/home` which means it **always** prints the home menu before running any command. I was curious why this addition was made.

Reply:
Can you comment on this @DidierRLopes (I assume you made it) just so that we don't merge something that has a clear purpose outside of `exe`.

Reply:
I believe the idea behind this was so that you could do
`stocks/economy/crypto/` which is technically

`stocks/home/economy/home/crypto/`

However this functionality still seems to work without this line

Reply:
Aha but then what most likely doesn't work is doing `/economy` within `stocks`?

Reply:
So this still does seem to work.  I can't quite figure out when this line of code would be hit.  I thought we already split `an_input` so it would never be `/cmd`, just `/` or `cmd`.  @Chavithra @montezdesousa @hjoaquim ?

```
2023 Jan 25, 15:40 (🦋) /stocks/options/ $ /crypto




╭───────────────────────────────────────────────────────────────────────────────────────────────────── Cryptocurrency ──────────────────────────────────────────────────────────────────────────────────────────────────────╮
│                                                                                                                                                                                                                           │
│     load               load a specific cryptocurrency for analysis                     [YahooFinance, CoinGecko, CCXT]                                                                                                    │
│     find               find coins                                                      [CoinGecko, CoinPaprika, Binance, Coinbase, YahooFinance]
```

Reply:
Hi, can one of you please add some clarification if we can now merge this one in. @Chavithra @montezdesousa @hjoaquim @jmaslek

Reply:
I can't break the code in any way so I am not sure if this line was necessary